hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CryptoCreateValidator.java

```diff
@@ -82,7 +83,10 @@ private void validateKeyAndAliasProvidedCase(
             @NonNull final CryptoCreateWithAliasConfig config,
             @NonNull final ReadableAccountStore readableAccountStore) {
         validateTrue(config.enabled(), NOT_SUPPORTED);
-        attributeValidator.validateKey(op.keyOrThrow());
+        // TODO - to match the mono-service synthetic hollow account creation, we need an empty key list here
+        if (!isEmpty(op.keyOrThrow())) {
+            attributeValidator.validateKey(op.keyOrThrow());
+        }
```

Comment:
hmm I don't see it [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/validators/CryptoCreateChecks.java#L163) , Is it added somewhere else ?

Reply:
We have a problem here because the synthetic `TransactionBody` that `mono-service` uses is not technically a valid HAPI `CryptoCreateTransactionBody`. 😒 

But I think with the new `AccountID payer` override, this could be changed to e.g.
```
// Skip this check the body is missing its TransactionID (implying it has been dispatched from the ContractService)
if (body.hasTransactionID()) {
    attributeValidator.validateKey(op.keyOrThrow());  
}
```